src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.service;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+import roomescape.dao.ReservationDao;
+import roomescape.dao.TimeDao;
+import roomescape.domain_entity.Id;
+import roomescape.domain_entity.Reservation;
+import roomescape.domain_entity.ReservationTime;
+import roomescape.dto.ReservationRequestDto;
+import roomescape.dto.ReservationResponseDto;
+
+@Component
+public class ReservationService {
+
+    private ReservationDao reservationDao;
+    private TimeDao timeDao;
+
+    public ReservationService(ReservationDao reservationDao, TimeDao timeDao) {
+        this.reservationDao = reservationDao;
+        this.timeDao = timeDao;
+    }
```

Comment:
```suggestion
    private final ReservationDao reservationDao;
    private final TimeDao timeDao;

    public ReservationService(ReservationDao reservationDao, TimeDao timeDao) {
        this.reservationDao = reservationDao;
        this.timeDao = timeDao;
    }
```
생성자 주입이 되었다면 불변하게 유지할 수 있겠네요.